include/crocoddyl/core/codegen/action-base.hpp

```diff
@@ -0,0 +1,119 @@
+///////////////////////////////////////////////////////////////////////////////
+// BSD 3-Clause License
+//
+// Copyright (C) 2020, LAAS-CNRS
```

Comment:
Please put the license of Inria here, as the code has been in part copy-pasted.

Reply:
oh I didn't realize this was already added! I'll remove this file for this PR.